content/momentum/changelog/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-lastUpdated: '04/08/2020'
-title: 'Momentum Changelogs'
-description: 'Momentum Changelogs'
+lastUpdated: "04/08/2020"
+title: "Category File"
+type: "custom"
+name: "Momentum Changelogs"
+description: "Momentum Changelogs"
 ---
 
-- [Momentum 4.x Changelogs](/momentum/changelog/4)
-- [Legacy Momentum Changelogs](/momentum/changelog/legacy)
-
----
+* [Momentum 4.x Changelogs](/momentum/changelog/4)
+* [Legacy Momentum Changelogs](/momentum/changelog/legacy)
```

Comment:
This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files